rcl/test/rcl/test_publisher.cpp

```diff
@@ -64,19 +63,10 @@ class CLASSNAME (TestPublisherFixture, RMW_IMPLEMENTATION) : public ::testing::T
 TEST_F(CLASSNAME(TestPublisherFixture, RMW_IMPLEMENTATION), test_publisher_nominal) {
   rcl_ret_t ret;
   rcl_publisher_t publisher = rcl_get_zero_initialized_publisher();
-  const rosidl_message_type_support_t * ts = ROSIDL_GET_MSG_TYPE_SUPPORT(std_msgs, msg, Int64);
-  // TODO(wjwwood): Change this back to just chatter when this OpenSplice problem is resolved:
```

Comment:
@wjwwood FYI I couldnt reproduce the issue so I restored `chatter` and removed the TODO